src/main/scala/dsptools/numbers/chisel_types/DspRealTypeClass.scala

```diff
@@ -118,9 +118,13 @@ trait DspRealReal extends DspRealRing with DspRealIsReal with ConvertableToDspRe
   override def fromInt(n: Int): DspReal = super[ConvertableToDspReal].fromInt(n)
   override def fromBigInt(n: BigInt): DspReal = super[ConvertableToDspReal].fromBigInt(n)
   def intPart(a: DspReal): SInt = truncate(a).toSInt()
-  // TODO: Implement? Will it ever be used? instead of div2?
-  def shr(a: DspReal, n: Int): DspReal = throw DspException(">> for DspReal is unimplemented")
-  def shr(a: DspReal, n: UInt): DspReal = throw DspException(">> for DspReal is unimplemented")
+  def shr(a: DspReal, n: Int): DspReal = div2(a, n)
```

Comment:
I forgot if there was a reason I had shr in DspRealReal instead of BinaryRepresentationDspReal... Maybe this should be moved up now?